README.md

```diff
@@ -14,23 +14,32 @@ You can see a working version of this bot in the GameUpscale discord server here
 
 ## Setup
 
-### Configuration
+### Installation
+
+Setup a python virtual environment: `py -m venv venv`

```

Comment:
I would maybe mention that setting up a virtual environment is optional, since you can just use your normal python env if you want (that's what I've been doing). 

Also, thanks for this!

Reply:
Just updated it with your suggestion, good call!